spec/features/bulkrax_import_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Bulkrax import', clean: true, perform_enqueued: true do
+  let(:user) { User.new(email: 'test@example.com') }
+  let(:account) { create(:account) }
+  # let(:admin_set) { create(:admin_set) }
+  let(:importer) do
+    create(:bulkrax_importer_csv,
+           user: user,
+           #  admin_set_id: admin_set.id,
+           field_mapping: field_mapping,
+           parser_klass: "HykuAddons::CsvParser",
+           parser_fields: { 'import_file_path' => import_batch_file },
+           limit: 0)
+  end
+  let(:import_batch_file) { 'spec/fixtures/csv/pacific_articles.csv' }
+  let(:field_mapping) do
+    {
+      # "title" => {"from"=>["title"], "split"=>true, "parsed"=>false, "if"=>nil, "excluded"=>false},
```

Comment:
Outstand commented code left here.

Reply:
I had left this in for a reason before but I can't recall what reason that is now.  I'll either add a comment or remove it in the next PR.